src/app/components/audio-recordings/pages/analysis-results/analyses-results.component.ts

```diff
@@ -0,0 +1,193 @@
+import { AudioRecording } from "@models/AudioRecording";
+import { Observable, map, takeUntil } from "rxjs";
+
+const audioRecordingKey = "audioRecording";
+const projectKey = "project";
+const regionKey = "region";
+const siteKey = "site";
+
+export const rootPath = "/analysis_jobs/system/results/";
+
+@Component({
+  selector: "baw-analyses-results",
+  templateUrl: "analyses-results.component.html",
+  styleUrls: ["analyses-results.component.scss"],
+})
+export class AnalysesResultsComponent extends PageComponent implements OnInit {
+  public constructor(
+    public api: AnalysisJobItemResultsService,
+    public route: ActivatedRoute
+  ) {
+    super();
+  }
+
+  public rows$: Observable<ResultNode[]>;
+  private readonly routeData = this.route.snapshot.data;
+  public audioRecording: AudioRecording = this.routeData[audioRecordingKey]?.model;
+  private rows = Array<ResultNode>();
+  private rootItemPath: string;
+
+  public get analysisJob(): AnalysisJob {
+    // since this component is currently only used for default analysis jobs, we can hard code the analysis job
+    return new AnalysisJob({
+      id: "system",
+      name: "system",
+    });
+  }
+
+  public ngOnInit() {
+    this.rootItemPath = `${rootPath}${this.audioRecording.id}/`;
+
+    const rootItem = new AnalysisJobItemResult({
+      name: "",
+      path: this.rootItemPath,
+      type: "directory"
+    });
+
+    const rootNode = {
+      result: rootItem,
+    };
+
+    this.rows = [rootNode];
+
+    this.rows$ = new Observable(subscribers => {
+      subscribers.next(this.rows);
+    });
+  }
+
+  /**
+   * Fetches the `AnalysisJobItemResult` model from the baw-api and converts it to to be compliant with the `ResultNode` interface
+   *
+   * @param model An incomplete result node that must include the result model name or id attribute
+   * @returns The complete `AnalysisJobItemResult` model of the requested item. If no model is not defined, the root path will be returned
+   */
+  public getItem(model?: ResultNode): Observable<AnalysisJobItemResult> {
+    const analysisJobId = this.analysisJob;
+    return this.api.show(model.result, analysisJobId, this.audioRecording.id);
+  }
+
+  private closeRow(model: ResultNode): void {
+    model.children?.forEach(child => {
+      // remove the child element from the rows
+      this.rows.splice(this.rows.indexOf(child), 1);
+
+      // close all the child elements of the row
+      this.closeRow(child)
+    });
+
+    model.open = false;
+  }
+
+  private openRow(model: ResultNode): void {
+    this.getModelChildren(model)
+      // since we have evaluated the children of the model, append this information to the model
+      // so that we don't have to re-query this information in the future from the API
+      .pipe(map(children => model.children = children))
+      .pipe(map(returnedValues => this.rows.splice(this.rows.indexOf(model) + 1, 0, ...returnedValues)))
```

Comment:
The sort algorithm might be possible by doing something along these lines:
1. Convert the paths to two arrays of ASCII/UTF value
2. Compare the two arrays value by value
  - If the value of `pathA < pathB` return -1, if the value of `pathA > pathB`, return 1, otherwise
  - If the character value of the nth character is equal, move to the next character and repeat

Reply:
Binning paths by path components and sorting by size was effective in creating a valid sort algorithm.